lenskit-core/specs/data-manifest.md

```diff
@@ -0,0 +1,111 @@
+# Data Manifest Schema
+
+This document describes the format of data manifests.
+
+Data manifests are JSON-style data.  Currently they are generally described in YAML files, but they may be described in other formats such as JSON.
+
+## Overall Layout
+
+A manifest file represents either a list or a map.
```

Comment:
given that you are defining things in terms of json style data types it might be worth making a stronger claim above.